dash_docs/chapters/external_resources/examples/addingExternalCSS.R

```diff
@@ -1,16 +1,25 @@
 library(dash)
 library(dashHtmlComponents)
 
+# external JavaScript files
 external_scripts <- list(
-  list('https://www.google-analytics.com/analytics.js'),
-  list(src = 'https://cdn.polyfill.io/v2/polyfill.min.js'),
-  list(src = 'https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.10/lodash.core.js')
+  src = 'https://www.google-analytics.com/analytics.js',
+  src = 'https://cdn.polyfill.io/v2/polyfill.min.js',
```

Comment:
```suggestion
  'https://www.google-analytics.com/analytics.js',
  'https://cdn.polyfill.io/v2/polyfill.min.js',
```
Anyway some of these changes conflict with #994 - maybe we just want to update that one to include examples with the full attribute syntax?

Reply:
Yeah, this branch has been held for weeks due to the `glue` error @HammadTheOne resolved, I apologize as this is a bit inconsistent given recent edits to #994. @alexcjohnson I'm happy to use the abbreviated syntax here too.